pkg/quota/resources.go

```diff
@@ -36,20 +40,16 @@ func Equals(a api.ResourceList, b api.ResourceList) bool {
 			return false
 		}
 	}
-	for key, value1 := range b {
-		value2, found := a[key]
-		if !found {
-			return false
-		}
-		if value1.Cmp(value2) != 0 {
-			return false
-		}
-	}
```

Comment:
why remove this, if a contains b, will alse return true

Reply:
a contains b (len(a) > len(b)) does not return true.

Reply:
It verifies if a and b equals

Reply:
lgtm

Reply:
Determine the key is equal to determine the value is equal? I think the definition of equality should be key equal and the value is equal. So need call value1.Cmp(value2),I think.

Reply:
Yes, and that's what it does now.

Reply:
U removed 
```
if value1.Cmp(value2) != 0 {	
	return false	
}
```

Reply:
Oh, That's a big bug, will revert it.

Reply:
No, it is hidden. see https://github.com/kubernetes/kubernetes/pull/64688/files/7feaec1008274263593495b279ee20809a6cfbf5#diff-2c2419fccdeffe90dff794bba3251a5dL35

Reply:
Oh, I see.IGNORE.

Reply:
looks good to me